launcher/InstanceImportTask.cpp

```diff
@@ -385,61 +388,83 @@ void InstanceImportTask::processFlame()
     connect(m_modIdResolver.get(), &Flame::FileResolvingTask::succeeded, [&]()
     {
         auto results = m_modIdResolver->getResults();
-        m_filesNetJob = new NetJob(tr("Mod download"), APPLICATION->network());
-        for(auto result: results.files)
-        {
-            QString filename = result.fileName;
-            if(!result.required)
-            {
-                filename += ".disabled";
+        //first check for blocked mods
+
+        QString text;
+        for(const auto& result: results.files.values()) {
+            if (!result.resolved || result.url.isEmpty()) {
+                text += QString("%1: <a href='%2'>%2</a><br/>").arg(result.fileName, result.websiteUrl);
             }
+        }
+        qWarning() << "Blocked mods found, displaying mod list";
+
+        auto message_dialog = new ScrollMessageBox(nullptr,
+                                                       tr("Blocked mods found") ,
+                                                       tr("The following mods were blocked on third party launchers.<br/>"
+                                                          "You will need to manually download them and add them to the modpack"), text);
+        m_blocked = true;
+        message_dialog->open();
+        message_dialog->activateWindow();
+        message_dialog->raise();
```

Comment:
Those can be just a `message_dialog->show()` if you make the dialog modal and set the parent afaik

Reply:
I can't really test that one out since i didn't notice that issue coming up